internal/entities/liquidity_provider/liquidity_provider.go

```diff
@@ -134,15 +136,15 @@ func ValidateConfiguration[T ConfigurationType](
 	}
 	if configuration == nil {
 		log.Warnf("Custom %s configuration not found. Using default configuration.", displayName)
-		return nil, errors.New("configuration not found")
+		return nil, ErrConfigurationNotFound
 	}
 	if err = configuration.CheckIntegrity(hashFunction); err != nil {
 		log.Errorf("Tampered %s configuration. Using default configuration. Error: %v", displayName, err)
 		return nil, err
 	}
 	if !signer.Validate(configuration.Signature, configuration.Hash) {
 		log.Errorf("Invalid %s configuration signature. Using default configuration.", displayName)
-		return nil, errors.New("invalid signature")
+		return nil, ErrInvalidSignature
```

Comment:
I like this. It's something that was missing in my previous PR. Can you please add also another sentinel error for the case of the tampered configuration and then remove all the logs? This is an entity and should not be showing any log.